src/Main.css

```diff
@@ -307,4 +307,106 @@ body {
 	background-color: #04213F;
 	border: 5px solid black;
 	border-radius: 8px;
-}
\ No newline at end of file
+}
+
+/* Button Styling */
+.btn {
```

Comment:
This would mean we have two definitions for the class .btn

I'm not a CSS expert, so I don't know if that is techincally allowed, but it confuses me, so I'm not a fan of it.